flutter/Makefile

```diff
@@ -149,18 +183,22 @@ backends/tflite-android:
 	rm -f android/app/src/main/jniLibs/arm64-v8a/libtflitebackend.so
 	cd .. && cp ${BAZEL_LINKS_DIR}bin/mobile_back_tflite/cpp/backend_tflite/libtflitebackend.so flutter/android/app/src/main/jniLibs/arm64-v8a/libtflitebackend.so
 
-ifeq (${ENABLE_BACKEND_EXAMPLE},1)
-backend_replace_example=\'example\',
-else
-backend_replace_example=
-endif
+# To add a new vendor backend, copy and uncomment this block and replace "EXAMPLE" and "example" with the vendor name (e.g. INTEL and intel)
+# See tflite backend below as an example
+#ifeq (${ENABLE_BACKEND_EXAMPLE},0)
+#backend_replace_example=
+#else
+#backend_replace_example=\'libexamplebackend\',
+#endif
```

Comment:
This way all backends will be enabled by default.
This isn't much of an issue because the app will silently skip ones that are missing but this kinda contradicts with the philosophy used for Android app, where each backend should be manually enabled.

Not sure how important this is, considering we need to refactor the entire flutter makefile in the near future.

Reply:
I was just following the current flutter approach. Please refactor as desired.

Reply:
Flutter makefile was a bit inconsistent on this.
Tflite backend is enabled by default and can be disabled.
"Example" backend was disabled by default, with `ifeq (${ENABLE_BACKEND_EXAMPLE},1)` check.
It would probably be better to place this into example, instead of tflite-style check. Though, I guess using different styles of checks can be a bit confusing, so maybe we should leave it the way it currently is.